src/sentry/api/endpoints/system_health.py

```diff
@@ -11,9 +13,14 @@ class SystemHealthEndpoint(Endpoint):
     permission_classes = (SuperuserPermission,)
 
     def get(self, request):
-        problems, checks = status_checks.check_all()
-
+        results = status_checks.check_all()
         return Response({
-            'problems': map(unicode, problems),
-            'healthy': checks,
+            'problems': map(
+                lambda problem: {
+                    'message': problem.message,
+                    'severity': problem.severity,
+                },
+                itertools.chain.from_iterable(results.values()),
+            ),
+            'healthy': {type(check).__name__: not problems for check, problems in results.items()},
```

Comment:
I'd like to change this return type to a mapping of status check name to problems (no problems implies healthy) unless anybody has a strong objection to it.